pkg/controller/volume/expand/expand_controller.go

```diff
@@ -280,14 +280,13 @@ func (expc *expandController) syncHandler(ctx context.Context, key string) error
 
 	volumePlugin, err := expc.volumePluginMgr.FindExpandablePluginBySpec(volumeSpec)
 	if err != nil || volumePlugin == nil {
-		msg := fmt.Errorf("didn't find a plugin capable of expanding the volume; " +
-			"waiting for an external controller to process this PVC")
+		msg := "waiting for an external controller to expand this PVC"
 		eventType := v1.EventTypeNormal
 		if err != nil {
 			eventType = v1.EventTypeWarning
 		}
-		expc.recorder.Event(pvc, eventType, events.ExternalExpanding, fmt.Sprintf("Ignoring the PVC: %v.", msg))
-		klog.Infof("Ignoring the PVC %q (uid: %q) : %v.", key, pvc.UID, msg)
+		expc.recorder.Event(pvc, eventType, events.ExternalExpanding, msg)
+		klog.Infof("waiting for an external controller to expand the PVC %q (uid: %q) : %v.", key, pvc.UID, msg)
```

Comment:
`waiting for an external controller to expand the PVC` is logged twice (it's already in `msg`)